src/collective/listingviews/browser/views/configure.zcml

```diff
@@ -48,12 +48,14 @@
         />
 
     <!-- to get macros for portlets -->
+<!--
```

Comment:
Again, if you are commenting something out, then either remove it once you have decided it is no longer needed, or leave a comment explaining why it has been commented out